examples/deploy_patterns/pyproject/pyproject_test.py

```diff
@@ -1,14 +1,19 @@
 import pathlib
 
 import flyte
+import requests
+import pandas
+
 
 env = flyte.TaskEnvironment(
-    name="pyproject_test",
+    name="pyproject_test_0",
     resources=flyte.Resources(memory="250Mi"),
-    image=flyte.Image.from_debian_base().with_uv_project(
-        pyproject_file=pathlib.Path("pyproject.toml"),
-        pre=True,
-    ),
+    image=(
+        flyte.Image.from_debian_base(registry="docker.io/nielsbantilan").with_uv_project(
+            pyproject_file=pathlib.Path("pyproject.toml"),
+            pre=True,
```

Comment:
You should be able to pass extra_args here
```suggestion
            pre=True,
            extra_args="--compile-bytecode"
```

Reply:
btw @pingsutw if you pass in `extra_args="--extra pandas"` it doesn't actually install the extra dependency (in the local image builder)